homeassistant/components/coinbase/__init__.py

```diff
@@ -29,6 +30,7 @@
             {
                 vol.Required(CONF_API_KEY): cv.string,
                 vol.Required(CONF_API_SECRET): cv.string,
+                vol.Optional(CONF_NATIVE_CURRENCY, default="USD"): cv.string,
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

For this integration, there is a PR open to migrating it. That PR needs to be completed before this feature can be added.

Any help getting the PR that adds the configuration flow merged, is always appreciated 👍 

../Frenck

Reply:
I see, my bad for not checking. I believe this feature is being added in the other PR anyways.

I'll check what I can do to help getting it merged ;)